README.md

```diff
@@ -18,8 +18,6 @@ npm install dotenv --save
 npm install dotenv-expand --save
 ```
 
-[![Become a Citizen](http://www.opensourcecitizen.org/badge)](http://www.opensourcecitizen.org/project?url=github.com/motdotla/dotenv-expand)
```

Comment:
Good call. I wasn't sure if this was still relevant because nothing ever loaded for me, but I think it was my adblocker at work and this domain is no longer relevant. It seems to be some sort of ad routing thing now.

Reply:
Though, I do wonder if this should be it's own, separate PR to keep things more atomic.

Reply:
Just so you are aware, this change specifically came when I rebased...

Reply:
If you look at the master, this and the package.json file updates already exist.

Reply:
👍